### PR TITLE
Add stocks command.

### PR DESCRIPTION
**Pull request by zaneneuschuler**
See title

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -5,7 +5,6 @@
   "files.trimTrailingWhitespace": true,
   "prettier.singleQuote": true,
   "prettier.trailingComma": "es5",
-  "editor.formatOnSave": true,
```
Comment (IanMitchell) [inline .vscode/settings.json]: Why did you remove this one?

File: package.json [hunk 2]
```diff
@@ -36,6 +36,7 @@
     "downsize": "^0.0.8",
     "esm": "^3.1.1",
     "form-data": "^2.3.3",
+    "grpc": "^1.20.3",
```
Comment (IanMitchell) [inline package.json]: Do we need this? Thought it was coming in as a sub dependency

File: src/bot/commands/stocks.js [hunk 3]
```diff
@@ -1,44 +1,138 @@
...
 export const info = {
   name: 'stocks',
-  description: '',
-  usage: '',
-  disabled: true,
+  description: 'Get some handy dandy stock info for your day trading!',
+  permissions: [Permissions.FLAGS.EMBED_LINKS],
+  usage: '```@Aquarius stocks <stock symbol>```',
 };
+function numberWithCommas(x) {
```
Comment (IanMitchell) [inline src/bot/commands/stocks.js]: Can remove this now right?

File: src/bot/commands/stocks.js [hunk 4]
```diff
@@ -1,44 +1,138 @@
...
+        aquarius.permissions.getRequestMessage(check.missing)
+      );
+      return;
+    }
+
+    try {
+      aquarius.loading.start(message.channel);
+      const response = await fetch(`${API}/${groups.sign}/${TOKEN}`);
+      const data = await response.json();
+      const embed = await createStocksEmbed(data);
```
Comment (IanMitchell) [inline src/bot/commands/stocks.js]: Don't need await here

File: src/bot/commands/stocks.js [hunk 5]
```diff
@@ -1,44 +1,138 @@
...
       message.channel.send(embed);
-      analytics.trackUsage('stocks', message);
+      aquarius.loading.stop(message.channel);
```
Comment (IanMitchell) [inline src/bot/commands/stocks.js]: Move this and the one in the catch outside of the try/catch

File: src/bot/commands/stocks.js [hunk 6]
```diff
@@ -1,44 +1,138 @@
...
+      log('Invalid permissions');
+      message.channel.send(
+        aquarius.permissions.getRequestMessage(check.missing)
+      );
+      return;
+    }
+
+    try {
+      aquarius.loading.start(message.channel);
+      const response = await fetch(`${API}/${groups.sign}/${TOKEN}`);
```
Comment (IanMitchell) [inline src/bot/commands/stocks.js]: Can we just use the raw string here instead of variables? Don't think they're reused, right?

File: src/bot/commands/stocks.js [hunk 7]
```diff
@@ -1,44 +1,138 @@
...
+function createStocksEmbed(data) {
+  const change = `Change ${
+    data.change >= 0
+      ? ':chart_with_upwards_trend:'
+      : ':chart_with_downwards_trend:'
+  }`;
+  const embed = new RichEmbed({
+    description: `*Stock Symbol: $${data.symbol}*`,
+    color: 16318579,
+    timestamp: `${fromUnixTime(data.latestUpdate / 1000)}`,
```
Comment (IanMitchell) [inline src/bot/commands/stocks.js]: Don't think we need the string interpolation here (and below in some places) just need `fromUnixTime(data.latestUpdate / 1000)`

File: src/bot/commands/stocks.js [hunk 8]
```diff
@@ -1,44 +1,138 @@
...
+      text: 'Data provided by IEX Cloud',
+    },
+    author: {
+      name: `${data.companyName} (Click here for more info!)`,
+      url: `https://finance.yahoo.com/quote/${data.symbol}/`,
+    },
+    fields: [
+      {
+        name: 'Open :bell:',
+        value: `${data.open}`,
```
Comment (IanMitchell) [inline src/bot/commands/stocks.js]: (like here)

File: src/bot/commands/stocks.js [hunk 9]
```diff
@@ -1,44 +1,138 @@
...
+        inline: true,
+      },
+      {
+        name: '52 Week Low :scream:',
+        value: `${data.week52Low}`,
+        inline: true,
+      },
+      {
+        name: 'Market Cap :moneybag:',
+<<<<<<< HEAD
```
Comment (IanMitchell) [inline src/bot/commands/stocks.js]: Merge conflict stuff

File: package.json (hunk 2 above)
Comment (zaneneuschuler) [reply package.json]: Without this, Firebase actively didn't work on my setup.

File: src/bot/commands/stocks.js (hunk 3 above)
Comment (zaneneuschuler) [reply src/bot/commands/stocks.js]: Yeah, forgot to remove it

**Comment by zaneneuschuler** [comment]
beep Desch#3091

**Comment by zaneneuschuler** [comment]
@IanMitchell Any update?

**Comment by IanMitchell** [comment]
Need some time, sorry - busy up until next week
